modules/xquare-user-interfaces/src/assets/icons/QuestionIcon.tsx

```
@@ -0,0 +1,28 @@
+  ...props
+}: QuestionIconProps) => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      height={`${size}px`}
+      viewBox="0 -960 960 960"
+      width={`${size}px`}
+      fill={color}
+      style={{ cursor: "help", ...props.style }}
+      {...props}
+    >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)